scripts/install_and_test_openqaoa_qiskit.sh

```diff
@@ -0,0 +1,23 @@
+# The script will stop if any intermediate file raises an error.
+set -e
+
+# TODO: Dynamically generate names (The order of installing and pushing is important here.)
+modulesList=("openqaoa-core" "openqaoa-qiskit")
+
+for entry in "${modulesList[@]}"; do
+    echo "processing src/$entry/setup.py"
+    cd src/$entry
+    pip install .
+    cd "../.."
+done
+
+for entry in "${modulesList[@]}"; do
+    echo "testing $entry"
+    cd src/$entry
+    pytest -n auto tests
```

Comment:
Yes! We have to rerun OQ-core tests for each combination of module install, this is because, some tests within core check for the plugins as well, thus, may potentially fail if the plugin has not been implemented properly. The tests in core go very fast anyways since they dont require qpu access or api access.

The alternative would be to either duplicate those tests in the plugin themselves but I don't think it makes much sense when rerunning the oq-core works just as well here.

Reply:
But can't we label those tests in a special way so that when the plugins run we only run those from core?

E.g. running all the vectorized tests for the Azure plugin seems redundant, no?

Reply:
Marking the tests makes sense as well.